fix: Provide Atlasmap getting started guide in README.md

jpav opened the pull request:
Fixes #509

apupier commented on README.md:
```diff
@@ -9,18 +9,48 @@
 This extension is providing a command to help development of [AtlasMap](http://docs.atlasmap.io/) transformations.
 
-### Preferences: Preferred Browser Type
+### Getting started
```
I think the "getting started" is a bit misleading in the context of the readme of VS Code AtlasMap.

To get started, it is more the "Open AtlasMap" command that seems important. And then pointing to the AtlasMap documentation to have more information on using the AtlasMap UI.

jpav replied:
I assume you mean in the original #509 issue? If so, yes, will do.

jpav replied:
I actually realized I put this all under the Features heading. I will move things around and change Getting Starting to Using AtlasMap.

apupier reviewed (changes requested):
- please put the jira issue number in commit message

lhein commented on README.md:
```diff
@@ -9,18 +9,48 @@
+    as described in the [AtlasMap documentation](http://docs.atlasmap.io/#running-atlasmap-with-apache-camel).
+-   Although the [Atlasmap documentation](https://docs.atlasmap.io/#exporting-mappings-to-an-atlasmap-catalog-file) states that
+    exported mappings are always exported to the Downloads folder, it now presents a file dialog to export to any accessible folder
+    on the file system. Also, the option to export to a catalog is now under the `AtlasMap` menu rather that via a toolbar
```
Rather than instead of rather that....looks fine to me

jpav commented on README.md:
```diff
@@ -9,18 +9,48 @@
+-   The AtlasMap documentation assumes it will be run in a standalone environment as opposed to being
+    embedded within VS Code, so some of the information may not apply. In particular, you can ignore the first step, "1. Quickly get
+    started using AtlasMap standalone".
+-   The look & feel of AtlasMap has changeed since the documentation was last updated and some of its screenshots have not yet been
```
I assume you mean in the original #509 issue? If so, will do.

apupier replied:
Not sure what is the most relevant to not forget it withotu bothering end users.
Maybe in the issue, it is enough.

apupier replied:
or maybe creating an issue mentioning to update the readme linked to the documentation update issue is even better. it will allow to have an opened issue in our project so easier to notice.

apupier commented on README.md:
```diff
@@ -30,8 +33,38 @@ To check if it has been started correctly, you can go to Output view and check f
+and save the artifact into your project.
 
 ## Prerequisites
 
-- Java 8+ must be installed on system path
+-   Java 8+ must be installed on system path
```
I think the prerequisites should be either at the top or at the bottom but it feels a bit strange in the middle

jpav replied:
Will do

apupier reviewed (changes requested):
a last small change suggestion.

Globally, it is greatly improving the readme.

apupier reviewed (changes requested):
please put the jira issue number in commit message

jpav commented:
Okay. I have no idea why this build is failing now.

apupier commented:
> I have no idea why this build is failing now.

me neither. it is very strange. the only difference that i spotted is a different circleci/node image version
https://app.circleci.com/pipelines/github/jboss-fuse/vscode-atlasmap/107/workflows/810f3d50-8ac9-4b7c-9a23-a5b1936d4095/jobs/121/parallel-runs/0/steps/0-0 9e422f7ff21351b5a6d257cf666b4947b905384907b331de6d3ed0bb0496b085

https://app.circleci.com/pipelines/github/jboss-fuse/vscode-atlasmap/109/workflows/96ace805-a4c7-4227-b461-a617ee13ae07/jobs/124/parallel-runs/0/steps/0-0 1a817546b1072e7cdcdeaaea5db3448c06b5eafa0d4318f18ed97c7b66730f95

There is same OS Ubuntu 18.04.04, same VS Code 1.52.2

apupier commented:
merged too fast, the PR wasn't approved.
you put the github issue number, not the jira issue number. When using the github issue, better to use #xxx notation to have automatic link. but too late to fix this time as it is merged

jpav commented:
Oh, sorry, I didn't know what the procedure was. In Atlasmap project, merge ability is only enabled after approval, so I assumed it was okay once I addressed all concerns. Regarding the issue number, there isn't a direct JIRA issue this addresses other than the epic itself, which contains many other issues. I think I can still update the PR though; I'll try.

apupier commented:
as it is for the commit message, it is already on the master. We do not force push on the master, this is too risky.

jpav commented:
k, I'll make sure not to commit prematurely in the future too.
